modules/validations/maprules.js

```diff
@@ -2,6 +2,9 @@ import { services } from '../services';
 
 
 export function validationMaprules() {
+    validation.type = 'maprules';
```

Comment:
This probably shouldn't be before the declaration of `validation`.

Reply:
oh thanks sorry - I was going quickly and didn't realize it wasn't like the others.